.ansible-lint

```diff
@@ -13,12 +13,16 @@ kinds:
   - playbook: "**/examples/*.yml"
 skip_list:
   - fqcn-builtins
+  - name[play]
+  - key-order[task]
 warn_list:
   - load-failure
+  - parser-error  # AnsibleParserError.
```

Comment:
Add `- playbook: "**/tests/get_coverage.yml" to `kinds` instead

Reply:
I think this will fix the `schema[tasks]` issue too (the error is because ansible-lint thinks `tests/get_coverage.yml` is a tasks file instead of a playbook

Reply:
and may fix the `schema[playbook]` issue as well